packages/vscode-extension/src/handlers.ts

```diff
@@ -874,14 +874,34 @@ function checkCoreNotEmpty(): Result<null, SystemError> {
 }
 
 export async function validateAzureDependenciesHandler(): Promise<string | undefined> {
+  if (commonUtils.checkAndSkipDebugging()) {
+    // return non-zero value to let task "exit ${command:xxx}" to exit
+    return "1";
+  }
+
+  try {
+    ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DebugEnvCheckStart);
```

Comment:
Do we need to try catch sendTelemetryEvent?

Reply:
We add try catch in case any exception... what is the recommended way to use `sendTelemetryEvent`?

Reply:
Confirm with @chagong , no need to try catch, shouldn't throw error inside sendTelemetryEvent.(Just a suggestion)